services/worker-manager/test/api_test.js

```diff
@@ -17,13 +17,14 @@ helper.secrets.mockSuite(testing.suiteName(), ['azure'], function(mock, skipping
 
   test('list providers', async function() {
     const {providers} = await helper.workerManager.listProviders();
-    assert.deepStrictEqual(providers.sort(), [
+    assert.deepStrictEqual(providers, [
```

Comment:
Why no sort? If the default ordering changes in a different node version or something this test will fail I think?

Reply:
The sort doesn't work because the items are objects and they aren't being sorted by key - so order matters either way here